packages/morph/src/morph.js

```diff
@@ -358,16 +358,16 @@ class DomManager {
     }
 
     traversals = {
-        'first': 'firstElementChild',
-        'next': 'nextElementSibling',
+        'first': 'firstChild',
+        'next': 'nextSibling',
```

Comment:
This one is the relevant change

Reply:
So, I think you're right in identifying the problem, however, this might not be the place. Sorry for the not-amazingly-readable code here, but here's my intent with these traversals:

By default all the traversals should operate on full Element nodes, unless you chain on the `.nodes()` method, in which case, all the traversal methods will be replaced with Node specific ones.

So if there's a place in the morph codebase that uses Node traversal in one place and Element in another, we should fix it upstream by adding `.nodes()` or subtracting `.nodes()`

Does that make sense?

Let me know if I'm the one with the miss-understanding and as always thank so much for your very high quality work on Alpine ❤️

Reply:
I think I've initially tried removing the nodes() method but some tests started failing because it wasn't teleporting the nodes anymore or something like that. I need to have a look again. 👍

Reply:
Gotcha. Yeah, that makes sense now. I've updated the PR to fix the issue in the way you suggested.